jobs/haproxy/spec

```diff
@@ -148,7 +148,8 @@ properties:
       A flag denoting the use of additional certificates from external sources.
       If set to true the contents of an external crt-list file located at `ha_proxy.ext_crt_list_file` are
       added to the crt-list described by the `ha_proxy.crt_list` property. Please be aware that reloading external certificates does only work if haproxy is
-      started in daemon mode. So this feature does not work if "haproxy.syslog_server" is set to "stdout".
+      started in daemon mode. So this feature does not work if "haproxy.syslog_server" is set to "stdout". If using this feature but not using internal certs,
```

Comment:
Hi @dom it definitely works because this test proves that it works :) 
https://github.com/cloudfoundry-incubator/haproxy-boshrelease/blob/2d558dadfa4cd52eda011b3c9f173126db6f645c/acceptance-tests/https_ext_crt_list_test.go#L246

It works because `[]` in Ruby evaluates to truthy so setting it to empty array ensures that all blocks in the templates where we are checking `crt_list` are executed.

Reply:
(That test has no internal certs but the request is correctly handled with the external cert)